From 8b5152d931b77c6fc35e66dd9a9d314ca306c217 Mon Sep 17 00:00:00 2001
From: hoahuynh <hoa.huynh.ud@renesas.com>
Date: Thu, 30 Jul 2020 13:52:11 +0700
Subject: [PATCH 438/455] arm64: dts: renesas: r8a774a3: Add USB3.0 peripheral
 node

This patch adds USB3.0 peripheral channel 0 device node
for RZ/G2M v3.0 (R8A774A3) SoC.

Signed-off-by: hoahuynh <hoa.huynh.ud@renesas.com>
---
 arch/arm64/boot/dts/renesas/r8a774a3.dtsi | 8 +++++++-
 1 file changed, 7 insertions(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/renesas/r8a774a3.dtsi b/arch/arm64/boot/dts/renesas/r8a774a3.dtsi
index fefcdb8..5ebb34b 100644
--- a/arch/arm64/boot/dts/renesas/r8a774a3.dtsi
+++ b/arch/arm64/boot/dts/renesas/r8a774a3.dtsi
@@ -1946,8 +1946,14 @@
 		};
 
 		usb3_peri0: usb@ee020000 {
+			compatible = "renesas,r8a774a3-usb3-peri",
+				     "renesas,rcar-gen3-usb3-peri";
 			reg = <0 0xee020000 0 0x400>;
-			/* placeholder */
+			interrupts = <GIC_SPI 104 IRQ_TYPE_LEVEL_HIGH>;
+			clocks = <&cpg CPG_MOD 328>;
+			power-domains = <&sysc R8A774A3_PD_ALWAYS_ON>;
+			resets = <&cpg 328>;
+			status = "disabled";
 		};
 
 		ohci0: usb@ee080000 {
-- 
2.7.4

